src/main/java/liquibase/ext/mongodb/statement/AbstractRunCommandStatement.java

```diff
@@ -41,13 +42,21 @@
 
     @Override
     public void execute(final MongoConnection connection) {
-        run(connection);
+        Document response = run(connection);
+        checkResponse(response);
     }
 
     public Document run(final MongoConnection connection) {
         return connection.getDatabase().runCommand(command);
     }
 
+    /**
+     * Check the response and throw an appropriate exception if the command was not successful
+     * @param responseDocument the response document
+     * @throws MongoException a MongoException to be thrown
+     */
+    abstract void checkResponse(Document responseDocument) throws MongoException;
+
```

Comment:
This should be `abstract` to ensure that implementations consider how to check the responses (i.e. providing a default no-op would not be suitable).